pkgs/by-name/vi/vivaldi/package.nix

```diff
@@ -235,6 +235,7 @@ stdenv.mkDerivation rec {
     mainProgram = "vivaldi";
     maintainers = with lib.maintainers; [
       marcusramberg
+      max06
```

Comment:
Note that you need to add yourself in the maintainer list first

https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md#how-to-become-a-maintainer

Reply:
See pr description 👍🏼 (PR is waiting for merge)